### PR TITLE
Replace request with axios

### LINKED ISSUE #204: Replace `request` lib?
Request is really bloated, and since it seems to only be used here to make a simple POST request, we can easily replace it with something lighter like got or axios. See https://bundlephobia.com/result?p=request@2.88.0

### PR DESCRIPTION
**Pull request by cktang88**
Fix #204 
Uses: https://bundlephobia.com/result?p=axios@0.19.0
Should save around **170kb** gzipped

### CODE REVIEW

**Comment by cktang88** [comment]
@XhmikosR review?

File: lib/sendToCoveralls.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 'use strict';
+// @ts-check
```
Comment (XhmikosR) [inline lib/sendToCoveralls.js]: This isn't needed

File: test/sendToCoveralls.js [hunk 2]
```diff
@@ -61,7 +63,7 @@ describe('sendToCoveralls', () => {
 
     // set up mock process.stdout.write temporarily
     const origStdoutWrite = process.stdout.write;
-    process.stdout.write = function(string, ...args) {
+    process.stdout.write = function (string, ...args) {
```
Comment (XhmikosR) [inline test/sendToCoveralls.js]: This should be reverted

**Comment by XhmikosR** [comment]
@cktang88 my main concern is if everything will work fine in case of exceptions.

/CC @nickmerwin

**Comment by XhmikosR** [comment]
Also, TBH I'm not sure I'd go with axios. Maybe node-fetch or got (8.x for Node.js 6.x support) would be a better choice.

**Comment by XhmikosR** [comment]
@nickmerwin this can be closed

**Pull request closed without merging**